### ISSUE TITLE
Python3 compatibility

### ISSUE THREAD

**Issue by dadosch** [apollo-ng/PiGI#31]
Python2 will be end of Line next year, (1. Januar 2020).

**Comment by dadosch** [comment]
all python depencies are availble from apt raspbian repos except python3-gevent-websocket

**Comment by chron0** [comment]
pip install gevent-websocket should pull it for 3, does it not? probably best to test out in python venv (and generally spoken too) :)

**Comment by dadosch** [comment]
> pip install gevent-websocket should pull it for 3, does it not? probably best to test out in python venv (and generally spoken too) :)

Yes, since python3-gevent-websocket is only available in Debian unstable. (But I generally like to install things from the package manager when they are available since then I don't have to check multiple places for updates.)

When used with a venv, a setup.py could be used to manage dependencies.


(Btw: Is this table from the "diygeigercounter" project also applicable for the R10 potentiometer from the PiGI?)
![grafik](https://user-images.githubusercontent.com/12801390/65084308-62f62400-d99a-11e9-89dd-e15cbfce2faa.png)
